### ISSUE TITLE
Multi batchsize training. 

### ISSUE THREAD

**Issue by Capchenxi** [cure-lab/MagicDrive#101]
To whom it may concern,

I have a problem when I change the train_batch_size into 2 in configs/runner/default_t.yaml file. The error occurs because of the line below
https://github.com/cure-lab/MagicDrive/blob/cc9d9ae7931b2caf1ba2d304f460ba10e850fa31/magicdrive/networks/unet_addon_rawbox.py#L310C13-L310C43

I wonder if this line should be repeat_size = [repeat_size, 1] since the imported repeat_size as an integer is the batchsize, and 1 is the default uncond cam number.

**Comment by Capchenxi** [comment]
Or I wonder how to do multi-batchsize training for video generation since I still got errors when I fixed the line mentioned above. Thansk.

**Comment by flymin** [comment]
I do not think it is trivial to support multi-batch training, right now the video training only supports one video on one process. I sometimes use the "batch" dim as T, which is the frame number, to reuse the image generation code. I am sorry for the inconvenience. It may take some effort to support multi-batch training.
